releasenotes/notes/py38_end_of_support-fa1fdea6ea02b502.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
```

Comment:
This feels more like it should be an `upgrade` section where if the user is still using 3.8 they need to upgrade their environment to now use a newer version, at a minimum 3.9. A deprecation means the user can continue to do what they were doing before but that way will issue a warning for some alternate that works now and where the former way will get removed in the future. Here the support is gone already hence the user needs to upgrade things.

Reply:
I changed this in the latest commit.